### PR TITLE
Bug 1284413 - Update smart-home filter name from Websites to Web. r=l…

### PR DESCRIPTION
**Pull request by danhuang1202**
…chang

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b8b9f72627042f55aebf13eea7aac7ad62338ebc

File: tv_apps/smart-home/locales/home.en-US.properties [hunk 1]
```diff
@@ -4,6 +4,7 @@ tv=Live TV
 device=Device
 app=Application
 website=Website
```
Comment (luke-chang) [inline tv_apps/smart-home/locales/home.en-US.properties]: According to my `grep`, the id `website` should be able to be safely removed.

Comment (danhuang1202) [inline tv_apps/smart-home/locales/home.en-US.properties]: I would remove `website` in this patch.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e383b1953f13403a38765952c25c842bff0e2a8c

**Pull request merged**